pyproject.toml

```diff
@@ -214,12 +214,14 @@ split_arguments_when_comma_terminated = true
 [tool.tox]
 legacy_tox_ini = """
 [tox]
-envlist = py37
+envlist =
+    py{38,38,310,311}
```

Comment:
```suggestion
    py{38,39,310,311}
```

Reply:
Your eagle-eyed Dutchman eyes caught my typo again. This was a test, of course. 😇

Reply:
Well, since your recent barrage of slights regarding my supposed sloppiness, I have been just scouring your changes for any hint of a fuck-up, merely as a sordid form of retribution
![](https://media.giphy.com/media/EjtN90rolA18s/giphy.gif)

Reply:
I think we have a very healthy working relationship. ❤️